src/main/java/ecommerce/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package ecommerce.service.impl;
+
+import ecommerce.dao.UserDao;
+import ecommerce.lib.Inject;
+import ecommerce.lib.Service;
+import ecommerce.model.User;
+import ecommerce.service.UserService;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    @Inject
+    UserDao userDao;
+
+    @Override
+    public User create(User user) {
+        return userDao.createUser(user);
```

Comment:
How do you creating shopping cart for new user?

Reply:
let's leave it as it is

Reply:
nice